tests/utils/TestSetup.cpp

```diff
@@ -46,3 +46,10 @@ Snowflake::Client::Connection* TestSetup::connectionWithAutocommitFactory(const
     }
     return cnx;
 }
+
+std::string TestSetup::getDataDir()
+{
+  const std::string current_file = __FILE__;
```

Comment:
How did you locate the current file? __FILE__ refers to the file at compile time.

Reply:
Well, as long as tests are compiled and run on the same machine. It should be fine. Actually, if you run tests locally, `__FILE__` is the best way to get current file/directory. Otherwise, you will need to specify environment variable.

Reply:
ok, fair enough. Unlike the internal Jenkins build, the current Travis job includes both build and tests. It work unless we split the job.